### PR TITLE
!omen command

### PR DESCRIPTION
**Pull request by drusepth**
Responds to users with ominous bullshit........

screen shot 2017-12-23 at 12 30 18 am

### CODE REVIEW

**Comment by euank** [comment]
I have a bad feeling about this.

Want to squash the commits up and add em to the `lint` section in the package.json?

File: modules/omen.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const outros = [
+    "they'll whisper you your fate", "they'll sing your final song", "they'll bring great luck",
+    'they, too, have teeth', 'they lie in wait', 'they are watching', 'they can still see you',
+    'they know your IP', 'they come ever faster', "they'll be here soon", 'the clock ticks on',
+    'some things are inevitable', 'fate is imminent', "sometimes there's nothing you can do",
+    "it's up to you", 'you deserve it all', "we don't always get what we deserve", 'you can do it'
+  ];
+
+  const omen_templates = [
+    '-intros- -targets- -actions-.. -outros-.',
```
Comment (LinuxMercedes) [inline modules/omen.js]: Please for the love of cthulu put three dots in the ellipsis

**Comment by drusepth** [comment]
@LinuxMercedes I fixed those blasphemous ellipses.

@euank I'm not sure what you mean by adding to the `lint` section. I ran `npm run lint` and it only complained about other files. 

```
~/Code/opensource/EuIrcBot (drusepth-patch-1)$ npm run lint

> EuIrcBot@0.0.1 lint /Users/abrown/Code/opensource/EuIrcBot
> eslint $(git ls-files | grep -E '\.js$' | grep -E '^(bot.js|index.js|modules/bully.js|modules/featurerequest.js|modules/mastodon|modules/schedule|modules/nickserv.js|modules/test2.js)')


/Users/abrown/Code/opensource/EuIrcBot/modules/bully.js
  3:28  warning  Unexpected unnamed function  func-names

/Users/abrown/Code/opensource/EuIrcBot/modules/nickserv.js
  8:23  warning  Unexpected unnamed function  func-names

/Users/abrown/Code/opensource/EuIrcBot/modules/test2.js
  5:22  warning  Unexpected unnamed function  func-names

✖ 3 problems (0 errors, 3 warnings)

~/Code/opensource/EuIrcBot (drusepth-patch-1)$ git status
```

I'm happy to squash and force-push, but GitHub also provides a "Squash and merge" function we can use on the Merge pull request button. ⛄️

**Comment by euank** [comment]
@drusepth since the `lint` regex in `package.json` doesn't include `modules/omen`, this file sin't being checked by that command. That's what I meant.

> I'm happy to squash and force-push, but GitHub also provides a "Squash and merge" function we can use on the Merge pull request button

I prefer not using that squash-and-meg github feature since it requires me interpreting what the correct combination of messages is, I'm not sure how it deals with ownership, it prevents commit signing, and it's a very course granularity (can only squash down to one commit).

**Pull request merged**